client/my-sites/plans-features-main/index.jsx

```diff
@@ -393,6 +440,7 @@ export class PlansFeaturesMain extends Component {
 				<div className="plans-features-main__notice" />
 				{ ! plansWithScroll && this.renderToggle() }
 				{ plansWithScroll && this.renderFreePlanBanner() }
+				{ this.renderProductsSelector() }
```

Comment:
Let's move this below the query components; in an ideal world they should be rendered before rendering anything.

Reply:
Sounds good 👍